configs/mainnet.yaml

```diff
@@ -72,7 +72,7 @@ SECONDS_PER_ETH1_BLOCK: 14
 # 2**8 (= 256) epochs ~27 hours
 MIN_VALIDATOR_WITHDRAWABILITY_DELAY: 256
 # 2**8 (= 256) epochs ~27 hours
-SHARD_COMMITTEE_PERIOD: 256
+MIN_VALIDATOR_ACTIVE_PERIOD: 256
```

Comment:
This is a breaking change and can mess up clients that expect this key to exist in configs. If we merge this, client binaries will have to either:
- Update their expect config name, work with spec tests, break with existing user configs
- Not update, break with spec tests and work with existing user configs

Reply:
Let's not do it then.